### PR TITLE
Force unicode from translation

### PR DESCRIPTION
**Pull request by emord**
http://manage.dimagi.com/default.asp?159061#883992

https://docs.djangoproject.com/en/1.5/ref/unicode/#translated-strings

### CODE REVIEW

**Comment by benrudolph** [comment]
Could this potentially be an issue with all errors we raise that aren't wrapped `unicode` and are lazily translated? If so, we should probably code review for this in the future because I think I've seen this around quite a bit

**Comment by czue** [comment]
I think in most contexts this resolves fine - it's just when it ends up in an exception email message that nothing ever tries to coerce it to a string/unicode object.

**Comment by czue** [comment]
@nickpell confirming that the issue you were reporting was with the error message and not the root cause (apparently something to do with a conflicting subscription). because it looks like this PR only fixes the message

**Comment by nickpell** [comment]
@czue That's correct

**Comment by benrudolph** [comment]
Ah ok got it

File: corehq/apps/accounting/models.py [hunk 1]
```diff
@@ -1281,7 +1281,7 @@ def new_domain_subscription(cls, account, domain, plan_version,
         if date_end is not None:
             future_subscriptions = future_subscriptions.filter(date_start__lt=date_end)
         if future_subscriptions.count() > 0:
-            raise NewSubscriptionError(_(
+            raise NewSubscriptionError(unicode(_(
                 "There is already a subscription '%(sub)s' that has an end date "
```
Comment (millerdev) [inline corehq/apps/accounting/models.py]: Curious why you opted for `unicode(_(...))` rather than `_(u"...")`?

Comment (benrudolph) [inline corehq/apps/accounting/models.py]: I think this is to force the execution of the lazy object rather than to force it into unicode

Comment (millerdev) [inline corehq/apps/accounting/models.py]: Ah, I see `_` is a reference to `ugettext_lazy`. In this case you could use `ugettext` (and drop the `unicode` call) to do that.

**Pull request merged**